doc/sphinx/changes.rst

```diff
@@ -679,6 +679,33 @@ Miscellaneous
   (`#16725 <https://github.com/coq/coq/pull/16725>`_,
   by Jim Fehrle).
 
+Changes in 8.17.1
+~~~~~~~~~~~~~~~~~
+
+A variety of bug fixes and improvements to error messages, including:
+
+- **Fixed:**
+  coqdep generating wrong dependencies when looking for META files
```

Comment:
Thanks! This might be a bit more precise?
```suggestion
  coqdep used to generating absolute paths when looking for plugins and META files
```

Reply:
@rlepigre If possible, we could even add why this was a problem (e.g., "which created issues when building a Coq project with Dune...")?

Reply:
OK, I made an attempt at that be editing my suggestion above.

Reply:
The problem was when building plugins in "findlib" mode. The change actually fixes a coqdep bug where the path was non-sensical.

The absolute to relative path change is to workaround a dune bug where dune gets confused when an external path is a symlink.

So there is a fix + a workaround for the bug. IMHO saying that dune "expects" a relative path is not fully accurate as if it was not for the bug, dune wouldn't care about the path being absolute at all (and in fact in any build this happens all the time, for example for coqc)

Reply:
OK. Here is one additional attempt then:

>**Fixed:** coqdep generating wrong dependencies when looking for META files (when building in "findlib" mode) and work around issue in Dune by emitting relative paths.

If it can be improved, feel free to suggest further changes. But we should keep the changelog entry relatively short.

Reply:
Yes very good point Théo, maybe the wording best for users could be something like "Fixed: in some cases, coqdep emitted incorrect paths for META files which prevented dune builds for plugins from working correctly "

Reply:
IMVHO the relative path thing is pretty internal and only relevant to developers

Reply:
WDYT @rlepigre ?

Reply:
Yeah I agree! I like your last version.